resources/measures/HPXMLtoOpenStudio/resources/appliances.rb

```diff
@@ -1245,8 +1224,13 @@ def self.apply(model, unit, runner, num_settings, rated_annual_energy,
         plant_loop.addDemandBranchForComponent(water_use_connection)
       end
 
-      design_level = sch.calcDesignLevelFromDailykWh(daily_energy)
-      peak_flow = sch.calcPeakFlowFromDailygpm(daily_water)
+      temperature_sch = OpenStudio::Model::ScheduleConstant.new(model) # FIXME
+      temperature_sch.setValue(52.666)
```

Comment:
instead of using hardcoded value, maybe we can use the wh_setpoint variable, like before.